AUTHORS

```diff
@@ -3611,5 +3611,14 @@ Nicolas DUBIEN <github@dubien.org>
 richiemccoll <richie.mccoll@hotmail.co.uk>
 sinkhaha <1468709106@qq.com>
 Mert Can Altın <mertgold60@gmail.com>
+Jan Osusky <Jan.Osusky@iblsoft.com>
+Gabriela Gutierrez <gabigutierrez@google.com>
```

Comment:
Duplicate: Both entries in AUTHORS for Gabriela Gutierrez are for @gabibguti. Needs a .mailmap entry to consolidate.